src/uct/ib/base/ib_md.c

```diff
@@ -224,6 +224,11 @@ typedef struct {
     struct ibv_mr                 **mrs;
 } uct_ib_md_mem_reg_thread_t;
 
+typedef struct {
+    uct_ib_md_mem_reg_thread_t ctx;
+    int                        mr_count;
```

Comment:
maybe unsigned?

Reply:
this is unrelate to my PR, bad rebase, pushed a revert commit @iyastreb